jobs/build/sweep/Jenkinsfile

```diff
@@ -30,39 +30,41 @@ node {
     def version = params.BUILD_VERSION
     def major = version[0]
     doozerOpts = "--group openshift-${version}"
-    stage("Init") {
-        echo "Initializing bug sweep for ${version}. Sync: #${currentBuild.number}"
-        currentBuild.displayName = "${version} bug sweep"
+    sshagent(["openshift-bot"]) {
```

Comment:
I don't think anything in `sweep` needs `openshift-bot` creds -- it's looking at BZ and ET, not github.